docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,263 @@ Postprocessing
 
 .. currentmodule:: fairlearn.postprocessing
 
+Fairlearn currently supports one postprocessing technique that is based on the
+paper *Equality of Opportunity in Supervised Learning* [#3]_. Unlike other
+mitigation techniques :class:`ThresholdOptimizer` is built to satisfy the
+specified fairness criteria exactly and with no remaining disparity.
+In many cases this comes at the expense of performance, for example, with
```

Comment:
The main point with usability is actually that it requires access to the sensitive feature at deployment time. We should mention it somewhere early on.

Reply:
Added that comment.